.github/workflows/validate-markdown.yml

```diff
@@ -0,0 +1,67 @@
+name: Validate Markdown
+
+on:
+  # Trigger the workflow on pull request
+  pull_request_target:
+    branches: 
+      - main
+    paths:
+      - '**.md'
+      - '**.ipynb'
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  check-broken-paths:
+    name: Check Broken Relative Paths
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Check broken Paths
+        id: check-broken-paths
+        uses: john0isaac/action-check-markdown@v1.0.6
```

Comment:
Do you have a latest tag?

Reply:
Same comment for the others.

Reply:
Nope, v1.0.6 don't worry the workflow rarely gets updated as it's very generic and can handle adding new features or functionality.
I'm installing the latest release from the [markdown-checker](https://github.com/john0isaac/markdown-checker) inside the workflow which means that if there are any problems with the code it can be fixed without the need to update the workflow.

If you still want the latest workflow we can use this 
`john0isaac/action-check-markdown@main` which insures that the workflow is always being executed from main branch not a release.

Reply:
using `john0isaac/action-check-markdown@latest` doesn't work.

![image](https://github.com/Azure-Samples/azure-search-openai-demo/assets/64026625/f24fb8a4-70da-4cc9-bc61-7489e1c78f15)

using `john0isaac/action-check-markdown@main` works.
![image](https://github.com/Azure-Samples/azure-search-openai-demo/assets/64026625/3679f510-a66a-438c-9376-41ef73939f67)

@tonybaloney let me know how you'd like to proceed do you prefer using @main or using @v1.0.6?

Reply:
Specific version is fine, it's just another thing to patch. If it doesn't change often it doesn't matter